main/getting-started/ertp-introduction.md

```diff
@@ -98,18 +109,22 @@ are not all the same, and it matters if you get third row center or
 second balcony far left  (and affects what you're willing to trade for
 it). 
 
-## Amount Math
+## AmountMath
 
-Issuers must be able to deposit and withdraw assets from a purse. This
+`Issuers` must be able to deposit and withdraw assets from a `purse`. This
 requires being able to add and subtract digital assets. They use a set
-of `amountMath` functions, which are aided by `mathHelpers` functions.
+of `amountMath` functions.
 
 In addition to math operations, `amountMath` functions check on their
-arguments' brands, throwing an error if the wrong brand was used.
+arguments' `brands`, throwing an error if the wrong `brand` was used.
+
+`amountMaths` only work on assets of their associated `brand` and its `issuer.
+There can be many copies of the `amountMath` for a particular `brand` and
+its `issuer`. 
 
 ## Next Steps
 
-If Getting Started, you should go to the [Introduction to Zoe](https://agoric.com/documentation/getting-started/intro-zoe.html).
+If you are Getting Started, you should go to the [Introduction to Zoe](https://agoric.com/documentation/getting-started/intro-zoe.html).
```

Comment:
Can we please use relative links so that the version running locally also links locally?
```suggestion
If you are Getting Started, you should go to the <router-link to="/getting-started/intro-zoe.html">Introduction to Zoe</router-link>.
```